src/save_gymmick.c

```diff
@@ -1,21 +1,21 @@
 #include "global.h"
 #include "gymmick.h"
 
-void SavGymmick_Clear(struct Gymmick *gymmick) {
+void Save_Gymmick_Clear(struct Gymmick *gymmick) {
     MI_CpuClear8(gymmick, sizeof(struct Gymmick));
 }
 
-union GymmickUnion *SavGymmick_Init(struct Gymmick *gymmick, int kind) {
-    SavGymmick_Clear(gymmick);
+union GymmickUnion *Save_Gymmick_Init(struct Gymmick *gymmick, int kind) {
+    Save_Gymmick_Clear(gymmick);
     gymmick->kind = kind;
     return &gymmick->data;
 }
 
-union GymmickUnion *SavGymmick_AssertMagic_GetData(struct Gymmick *gymmick, int kind) {
+union GymmickUnion *Save_Gymmick_AssertMagic_GetData(struct Gymmick *gymmick, int kind) {
     GF_ASSERT(gymmick->kind == kind);
     return &gymmick->data;
 }
 
-int SavGymmick_GetType(struct Gymmick *gymmick) {
+int Save_Gymmick_GetType(struct Gymmick *gymmick) {
```

Comment:
What does `Save` refer to here exactly? Why is this file called `save_gymmick.c`?

Reply:
I feel like this isn't `Save` at all and it's just `Gymmick`, `Save` usually means that the functions actually directly access the Save Array but none of the functions here do.

Reply:
out of scope

Reply:
this is a save PR it isnt outta scope

Reply:
this pr is renaming savedata to saveData, classifying whether Gymmick belongs under Save is out of scope (basically I'd prefer to do this in another PR)